packages/ui-bindings/AgentRelationship/agentRelationshipInterface.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * A This file defines common interfaces for the AgentRelationship object
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2018-02-02
+ */
+
+import { gql } from "react-apollo";
+import { Agent } from "../Agent/agentInterface";
+import { AgentRelationshipRole } from "../AgentRelationshipRole/AgentRelationshipRole";
+
+export const agentRelationshipInterface = gql`
+fragment agentRelationshipInterface on AgentRelationship {
+  id
+  subject{
+    id
+  }
+  object{
+    id
+  }
+  relationship{
+    id
+  }
+}`;
+
+export interface AgentRelationship {
+  id: number
+  name: Agent
+  symbol: Agent
+  relationship: AgentRelationshipRole
```

Comment:
I notice that you're referring to the exact object type rather than just 'Object'. Is this simply because these objects have already been implemented in the project? And if so, should we plan to later revise all Objects to be their appropriate type?

Reply:
yes.